src/App.tsx

```diff
@@ -117,37 +120,53 @@ export const checkWinCondition = (b: typeof reactBoard): WinState => {
 
 }
 
-function renderMove(move: string, idx: number) {
-  if (move === "X") {
-    return (<div>X</div>)
-  }
-  if (move === "O") {
-    return (<div>O</div>)
-  }
-  if (move === "") {
-    return (<div>space</div>)
-  }
+type MoveProps = {
+  board: Board
+  setBoard: React.Dispatch<React.SetStateAction<Board>>
+  move: string
```

Comment:
I THINK there's some duplication here as the "move" value should be referencable via the board + rowIndex + mvIndex